cmd/livepeer/livepeer.go

```diff
@@ -663,6 +667,21 @@ func main() {
 		if server.AuthWebhookURL, err = getAuthWebhookURL(*authWebhookURL); err != nil {
 			glog.Fatal("Error setting auth webhook URL ", err)
 		}
+
+		// Set up verifier
+		if *verifierAddr != "" {
```

Comment:
For other external URLs entered as flags we prefix the URL with `https://` if it doesn't have a http protocol prefix, do you think it would be a good idea to add it here as well, just to have a uniform approach. Otherwise it might lead to confusion when some flags require the prefix and some don't.

Reply:
This probably needs to be renamed to `verifierUrl` or something [1], including the CLI flag. Most of our other `*Addr` flags technically only take a `host:port` combination, but this is more of a full URL, including the scheme and path such as `http://host:port/path/to/verifier`

[1] For our other external callback URLs (`orchWebhookUrl` and `authWebhookUrl`), we actually fail out if the prefix isn't http/https, rather than trying to correct it. Probably should do similar here - in fact, the below could use some refactoring.

https://github.com/livepeer/go-livepeer/blob/7f441276ac75c235e6578157e462a7ec5d82e275/cmd/livepeer/livepeer.go#L772-L800

Reply:
failing when the prefix isn't included sounds good to me !

Reply:
Made the following changes [1]

* Refactored to share webhook URL parsing code (should be no functional changes) 8e9062d393ad69b38b232e3e54179540d992f022

* Updated to use `-verifierUrl` as the flag name, following the scheme (and capitalization 😦) of our external webhook URLs in e62923951858d966623c3e855055039535b18f6d

* Additional CLI tests in b5665a81609ee648de6796680acd76506b1faa6e

[1] Went ahead and fixed up the commits locally rather than pushing a separate fixup commit for review since the timeline here is tight and there would have been merge conflicts due to a few commits changing the same area of code in `livepeer.go`

Reply:
Thanks ! Changes look good 👍